Fixed toogle full screen

gemi254 opened the pull request:
Resize player window by clicking on the video stream or pressing the icon near close stream (x),  
1. Original size 
2. Maximized 
3. Fullscreen with aspect 
4. Fullscreen

s60sc commented:
Hi
On Chrome, the image window does not appear until a control button is pressed which may confuse new users

gemi254 commented:
Hi..
Fixed hide image window and changed default mode to `maximized` in order to work better with mobile phones
1. Maximized
2. Original size
3. Fullscreen with aspect
4. Fullscreen
